doc/source/multivisit.rst

```diff
@@ -3,4 +3,535 @@
+steps, these files are renamed based on the SkyCell to be generated in the current working directory.
+
+For reference, the original pipeline-assigned name has the format of:
+
+  `IPPPSSOOT_flc.fits`
+
+such as
+  * jcz906dvq_flc.fits
+  * icz901wpq_flc.fits
+
+The MVM filename defined for the **input exposures** follows the convention.
+
+  `hst_skycell-p<PPPP>x<XX>y<YY>_<instr>_<detector>_<filter>_<ipppssoo>_fl[ct].fits`
+
+where:
+  ==========  =========================================================================
+  Element     Definition
+  ==========  =========================================================================
+  <PPPP>      ProjectionCell ID as a zero-padded 4 digit integer
+  <XX>,<YY>   SkyCell ID within ProjectionCell as zero-padded 2 digit integers
+  <instr>     Name of HST instrument from **INSTRUME** header keyword
+  <detector>  Name of detector from **DETECTOR** header keyword
+  <filter>    Name of filter from **FILTER** or **FILTER1,FILTER2** header keyword(s)
+  <ipppssoo>  Pipeline-assigned **IPPPSSOO** designation from original input filename
+  fl[ct]      Suffix of either **flt** or **flc** from original input filename
+  ==========  =========================================================================
+
+This insures that each exposure gets renamed in a way that allows them to be easily identified with respect to the
+**output SkyCell layer** the exposure contributes to during MVM processing.
+
+
+Input Poller File
+------------------
+The MVM processing could simply combine whatever input files are present in the current working directory.  However,
+that may result in working with more than 1 SkyCell at a time which can, for some steps, end up requiring more memory
+or disk space than is available on the system.  Therefore, the code relies on an input `poller file` which specifies exactly
+what files should be processed at one time.  This ASCII CSV-formatted input `poller file` will only contain
+the names of exposures which
+overlap only a single SkyCell regardless of instrument, detector or any other observational configuration.
+
+Generating one of these input 'poller' files with the filename *skycell-p1889x07y19_mvm_input.txt* can be done using
+the same SkyCell dictionary defined earlier with the commands:
+
+.. code:: python
+
+    from drizzlepac.haputils import make_poller_files as mpf
+
+    with open('skycell-p1889x07y19_input.txt', 'w') as fout:
+        _ = [fout.write(f'{file}\n') for file in p1889_scells['skycell-p1889x07y19'].members]
+
+    mpf.generate_poller_file('skycell-p1889x07y19_input.txt',
+                             poller_file_type='mvm',
+                             output_poller_filename='skycell-p1889x07y19_mvm_input.txt',
+                             skycell_name='skycell-p1889x07y19')
+
+
+The full description of the function used to create this poller file can be found in the
+:ref:`Multi-visit Processing Code API documentation <multivisit_api.rst>`.
+
```

Comment:
In the view of the documentation I am using, the last portion of this line is not a link, and it should be.  I see you have this set up as a link, so it just be verified to be correct.

Reply:
Fixed in 3cd7f7e.